src/v3/src/transformer/password/transformReEnrollCustomPasswordExpiryWarning.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import Util from '../../../../util/Util';
+import {
+    ButtonElement,
+    ButtonType,
+    DescriptionElement,
+    IdxStepTransformer,
+    LinkElement,
+    TitleElement,
+  } from '../../types';
+  import { loc } from '../../util';
+
+  const getPasswordExpiryTitle = (daysToExpiry = -1): string => {
+    if (daysToExpiry > 0) {
+      return loc('password.expiring.title', 'login', [daysToExpiry]);
+    } else if (daysToExpiry === 0) {
+      return loc('password.expiring.today', 'login');
+    } else {
+      return loc('password.expiring.soon', 'login');
+    }
+  };
```

Comment:
I think what you could do is extract [this](https://github.com/okta/okta-signin-widget/blob/0.5/src/v3/src/transformer/password/transformExpiredPasswordWarningAuthenticator.ts#L26-L36) function into a utils file and use it here and in the original location?